sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -470,6 +472,13 @@ public static partial class MultipartResponse
         public static Azure.Response[] Parse(Azure.Response response, bool expectCrLf, System.Threading.CancellationToken cancellationToken) { throw null; }
         public static System.Threading.Tasks.Task<Azure.Response[]> ParseAsync(Azure.Response response, bool expectCrLf, System.Threading.CancellationToken cancellationToken) { throw null; }
     }
+    public partial class ProcessingContext
+    {
+        internal ProcessingContext() { }
+        public int AttemptNumber { get { throw null; } }
+        public System.Exception? LastException { get { throw null; } }
+        public System.DateTimeOffset OperationStartTime { get { throw null; } }
```

Comment:
Is this the time the caller invoked the service method?  The time the message was created?  How would this be used by a library author/library user?

Reply:
Yeah it would be the time the message was created which should be pretty close to when the method was called. The use case for this is if someone wants to create a policy that enforces an overall time limit on the operation (including retries).